sdk/include/psdk/tmschema.h

```diff
@@ -1247,6 +1247,91 @@ BEGIN_TM_PART_STATES(VERTTHUMB)
     TM_STATE(4, VTS, DISABLED)
 END_TM_PART_STATES()
 
+/* AEROWIZARD parts */
```

Comment:
I wonder whether it would be of interest to surround the NT6+ stuff within `#if (NTDDI_VERSION >= NTDDI_VISTA) ... #endif`